task_DEV-3.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;

```

Comment:
Are you sure all necessary files are added? Make temp clone from git and try to build project.